en/admin/admin_utils.rst

```diff
@@ -3343,6 +3344,119 @@ The following shows [options] available with the **cubrid vacuumdb** utility.
 
         cubrid vacuumdb -C demodb
 
+.. _flashback:
+
+flashback
+---------
+
+The **cubrid flashback** utility is used to get SQL statements to flashback a specific transaction, and can only be executed by the **DBA** user. The system parameter **supplemental_log** must be turned on, and flashback is supported only for DML executed after the **supplemental_log** is set. ::
```

Comment:
Is it just for "a" transaction though it will extract SQLs of several transactions at once?
```suggestion
The **cubrid flashback** utility is used to get SQL statements to flashback transactions within a specific time period, and can only be executed by the **DBA** user. The system parameter **supplemental_log** must be turned on, and flashback is supported only for DML executed after the **supplemental_log** is set. ::
```

Reply:
1. Information about multiple transactions within a specific time period is displayed in `summary phase`
2. and SQL statements are displayed only for one transaction selected by the user.

Reply:
It would be nice to include that flashback is provided for completed transactions (commit, abort). Because we don't know the level of readers who read the manual.

Reply:
Yes, I will add that description.